cli/tests/subdir/worker_error.ts

```diff
@@ -0,0 +1,5 @@
+function foo() {
+  throw new Error("foo");
```

Comment:
What happens with other built-in error classes like `TypeError`, `RangeError`, `SyntaxError`?  Do we need to send more than just the error message?

Reply:
The "message" being sent is `Uncaught <error.name>: <error.message>`. This is all that's sent to the host in browsers: 
![image](https://user-images.githubusercontent.com/29990554/96535668-380b8d00-128a-11eb-91d4-69aebfc0f8d2.png)

Reply:
Ok, cool!